src/components/Blog/BlogArticlesCollections/BlogArticlesCollections.tsx

```diff
@@ -0,0 +1,199 @@
+interface CategoryTabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  tag: TagAttributes;
+  tags: GetTagsResponse;
+  pagination: StrapiMetaPagination;
+  ariaLabel: string;
+  data: BlogArticleData[];
+}
+
+function CategoryTabPanel({
+  children,
+  index,
+  tag,
+  tags,
+  data,
+  pagination,
+}: CategoryTabPanelProps) {
+  const theme = useTheme();
+  const [pageTab, setPageTab] = useState(pagination.page);
+  const chunkedPages = chunkArray(data, pagination.pageSize);
+  console.log('chunkedPages', chunkedPages);
+  return (
+    <BlogArticlesCollectionsContainer>
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          flexDirection: 'column',
+        }}
+      >
+        <BlogArticlesBoardHeader>
+          <BlogArticlesBoardTitle variant="headerMedium">
+            {tag.attributes.Title}
+          </BlogArticlesBoardTitle>
+        </BlogArticlesBoardHeader>
+        {chunkedPages.map((page, pageIndex) => (
+          <CategoryPaginationPage pageTab={pageTab} index={pageIndex}>
+            {page.map((article, articleIndex: number) => (
+              <BlogArticleCard
+                styles={{
+                  display: 'inline-block',
+                  [theme.breakpoints.up('sm' as Breakpoint)]: {
+                    width: 'unset',
+                  },
+                }}
+                baseUrl={tags.url}
+                id={article.id}
+                key={`blog-articles-collection}-${index}-${articleIndex}`}
+                image={article.attributes.Image}
+                title={article.attributes.Title}
+                slug={article.attributes.Slug}
+                trackingCategory={TrackingCategory.BlogArticlesCollection}
+                content={article.attributes.Content}
+                publishedAt={article.attributes.publishedAt}
+                createdAt={article.attributes.createdAt}
+                tags={article.attributes.tags}
+              />
+            ))}
+          </CategoryPaginationPage>
+        ))}
+      </Box>
+      {
+        /* todo: enable pagination*/
+        pagination.pageCount > 0 ? (
+          <Pagination
+            isEmpty={pagination.pageCount <= 1}
+            page={pageTab}
+            setPage={setPageTab}
+            pagination={pagination}
+            categoryId={index}
+          />
+        ) : null
+      }
+    </BlogArticlesCollectionsContainer>
+  );
+}
+
+function chunkArray(array: any[], chunkSize: number): any[][] {
+  const result = [];
+
+  for (let i = 0; i < array.length; i += chunkSize) {
+    const chunk = array.slice(i, i + chunkSize);
+    result.push(chunk);
+  }
+
+  return result;
+}
+
+interface CategoryPaginationPageProps {
+  pageTab: number;
+  index: number;
+}
+
+const CategoryPaginationPage: React.FC<
+  PropsWithChildren<CategoryPaginationPageProps>
+> = ({ children, pageTab, index }) => {
+  return (
+    <ArticlesGrid hidden={pageTab !== index} active={pageTab === index}>
```

Comment:
some thing as above, do not load them into the dom if not shown to avoid increasing our page size + reducing perf

Reply:
Done